.github/workflows/master.yml

```diff
@@ -27,16 +27,11 @@ jobs:
         run: pnpm install
 
       - name: Lint
-        run: npx eslint --ignore-path .gitignore .
+        run: pnpm exec eslint --ignore-path .gitignore .
```

Comment:
by the way, why not use `pnpm run lint` here?

Reply:
Good call.